dashboard/test/models/survey_result_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class SurveyResultTest < ActiveSupport::TestCase
+  test "proper SurveyResult members" do
+    assert SurveyResult::ETHNICITIES["asian"] == "Asian"
+    assert SurveyResult::RESULT_ATTRS.include? "survey2016_ethnicity_asian"
+  end
```

Comment:
I'm not sure whether we have any standard way to test serialized_attrs when used in a model... that's about it.